allure-framework-modules/allure-webapi-module/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>allure-test-framework</artifactId>
+        <groupId>com.capgemini.ntc</groupId>
+        <version>2.0.0</version>
+    </parent>
+    <artifactId>mrchecker-webapi-module</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>MrChecker - WebApi - Module</name>
+    <description>MrChecker WebApi supports:
+        TODO
```

Comment:
It is not a good practise to leave TODO comment in code merged to develop branch. There is quite big chance that it stays forever.